extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js

```diff
@@ -72,7 +72,7 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 			provider + '.' + getGroup(slotName),
 			slotName.toLowerCase(),
 			device,
-			skin + '-' + pageType,
+			skin + '-' + (getContextTargeting().hasFeaturedVideo ? 'fv-' : '') + pageType,
```

Comment:
Can we extract it to separate method? `getLayout`? We're going to have a few more layouts and it will be helpful/more readable